index.rst

```diff
@@ -44,18 +44,589 @@
+
+.. note::
+   The Telescope and Site top-level CS document is LTS-136 :cite:`LTS-136`, and the Camera top-level CS document is LCA-280 :cite:`LCA-280`. Unlike those two documents, we do not define the Observatory Mount CS or the Azimuth CS, because these CS, as defined by LTS-136 :cite:`LTS-136` and LCA-280 :cite:`LCA-280`, actually contradict each other, in terms of which axis points north, which points west etc. We also avoid the use of terms such as ``elevation angle`` and ``altitude``. Instead, we always use ``zenith angle``, as defined above.
+
+
+################
+Zemax and PhoSim
+################
+
+We discuss the Zemax and PhoSim CS first, because these are relatively easier to define -
+they are self-consistent within their own framework.
+When we say Zemax CS, we refer to the global CS as used by
+`the official Rubin Observatory optical model <https://confluence.lsstcorp.org/display/SYSENG/As-built+optical+model>`__. The good news here is that when we worked with the PhoSim team at Purdue on the AOS simulations, we made sure that the PhoSim CS conforms to the project standard, at least externally, to the level that we care about while exercising AOS control.
+The Zemax/PhoSim CS is defined as,
+
+- The origin of Zemax/PhoSim CS overlaps with OCS origin, i.e., at the theoretical vertex of M1.
+- The +z axis of Zemax/PhoSim CS points from the sky to M1M3. It follows the direction of the incoming on-axis rays. This is opposite of the OCS +z axis.
+- The +y axis is the same as OCS +y axis.
+- The +x axis is the opposite of OCS +x axis
+
+.. figure:: /_static/zcs.png
+   :name: fig-zcs
+   :target: ../_images/zcs.png
+   :alt: The Zemax/PhoSim Coordinate System (ZCS)
+
+The Zemax/PhoSim Coordinate System (ZCS)
+
+.. code-block:: py
+
+   def zcs2ocs(x,y,z):
+       return -x,y,-z
+   def ocs2zcs(x,y,z):
+       return -x,y,-z
+
+The optical sensitivity matrix (senM) is derived using the Zemax optical model.
+Therefore, everything about the senM follows the ZCS. We were able to close the simulation loop with PhoSim, because we made PhoSim consistent with Zemax.
+With the actual hardware, we will need to convert all commands returned the AOS control into the proper CS of each component before they are applied.
+
+.. note::
+
+    Note that we apply the decenters and tilts in Zemax via ``Coordinate Breaks``. Mathematically the order of decenters and tilts matter. In Zemax, there is a ``order flag``. When it is set to 0, Zemax does the decenters first, then x-tilt, y-tilt, z-rotation. When the ``order flag`` is set to 1, Zemax does these in exact opposite order, so that users can easily go back to the original CS :cite:`Zemax13manual`. However, in the AOS context, we don't really care about these because the tilts are always small enough for the order not to make a difference. If this is not true, then the basic approach of taking the decenters and tilts of the hexapods as independent variable in the AOS control wouldn't be correct.
```

Comment:
When you say the tilts are small enough, can you specify how small it should be to fulfill this?

Reply:
well, it depends on how accurate you want it to be, i.e., what your tolerance is on the difference due to the ordering. I guess one can do some simple numerical experiments to get an idea (I played with it years ago). Or, the fact that we take them as independent variables and still can make AOS converge is a proof that the ordering does not matter for practical purposes.